analytic_engine/src/sst/parquet/async_reader.rs

```diff
@@ -198,12 +297,65 @@ impl<'a> Reader<'a> {
     }
 }
 
-#[derive(Debug)]
+/// Options for `read_parallely` in [Reader]
+#[derive(Debug, Clone, Copy)]
+struct ParallelismOptions {
```

Comment:
How about this
```
enum ParallelismOptions {
  Disable,
  Enable(max_row_groups_in_batch)
}
```

Reply:
I have removed `max_row_groups_in_batch` and revert the logic.
Because if we set the batch_size to integral multiple `num_rows_per_row_group`, a serious error will happen in the new logic due to the implementation of parquet reader.
So, we shouldn't support it now and should make it a todo.